### PR TITLE
Add Canadian Bandplan

### PR DESCRIPTION
**Pull request by Starman0620**

### CODE REVIEW

File: root/res/bandplans/canada.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    {
+      "name": "TV Channels 2-4",
+      "type": "broadcast",
+      "start": 54000000,
+      "end": 72000000
+    },
+    {
+      "name": "FM Broadcast",
+      "type": "broadcast",
+      "start": 76000000,
```
Comment (JoshuaKimsey) [inline root/res/bandplans/canada.json:143]: Though I am not familiar with Canadian radio bands, is this correct? I'm not sure I've ever seen FM commercial radio frequencies this low?

**Review by JoshuaKimsey — COMMENTED** [review]
Everything looks good! Just check on that FM commercial radio frequency and make sure it's correct.

File: root/res/bandplans/canada.json (hunk 1 above)
Comment (Starman0620) [reply root/res/bandplans/canada.json:143]: Honestly I've lived here my whole life and can't recall hearing of any stations at such low frequencies. However, it is clearly how the band is supposed to be according to the government websites I ripped the values from.

Sources:
https://www.ic.gc.ca/eic/site/smt-gst.nsf/vwapj/2014_Canadian_Radio_Spectrum_Chart.pdf/$file/2014_Canadian_Radio_Spectrum_Chart.pdf
https://www.ic.gc.ca/eic/site/smt-gst.nsf/eng/sf10759.html#t2

Comment (JoshuaKimsey) [reply root/res/bandplans/canada.json:143]: You are right, and apparently other countries go down to 76MHz as well, including Japan. I honestly don't know any normal radio that goes down that low, and here in the US that's an area we broadcast TV channels on. Maybe it's used for more specialty applications, like digital radio or something? Oh well, works for me! In that case, everything seems good!

**Review by JoshuaKimsey — APPROVED** [review]

**Pull request merged**